stdlib/ballerina-http/src/main/ballerina/http/failover-connector.bal

```diff
@@ -334,7 +334,7 @@ function performFailoverAction (string path, Request request, HttpOperation requ
     var mimeEntity = failoverRequest.getEntity();
     match mimeEntity {
         mime:Entity entity => requestEntity = entity;
-        mime:EntityError => io:println("mimeEntity null");
+        error => io:println("mimeEntity null");
```

Comment:
Can you check with Vijitha about this.. Printing something random doesn't sound right.